plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-fields-block/frontend.tsx

```diff
@@ -10,7 +10,7 @@ const FrontendBlock = ( {
 	children,
 	className,
 }: {
-	children: JSX.Element;
+	children: JSX.Element[];
```

Comment:
This is deprecated, I think we can use `ReactElement` instead of `JSX.Element`

when importing you should use it like this:

`import type { ReactElement } from 'react';` (the `type` keyword is important because you get an eslint error otherwise).